lib/chat/db/free_spaces.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Chat.Db.FreeSpaces do
+  @moduledoc "Free space in MB of databases"
+
+  alias Chat.Db.{InternalDb, MainDb, BackupDb, CargoDb}
+  alias Chat.Db.Maintenance
+
+  def get_all() do
+    spaces =
+      [InternalDb, MainDb, BackupDb, CargoDb]
+      |> Stream.map(&{&1 |> atomize_db_name(), &1 |> get_one()})
+      |> Enum.into(%{})
+
+    spaces
+    |> Map.put(:media_db, spaces.backup_db + spaces.cargo_db)
+    |> Map.drop([:backup_db, :cargo_db])
+    |> Enum.map(fn {key, value} ->
+      value =
+        case value do
+          0 -> "Not found"
+          value -> value |> bytes_to_MB()
+        end
+
+      {key, value}
+    end)
+    |> Enum.into(%{})
+  end
+
+  defp get_one(db) do
+    pid = Process.whereis(db)
+
+    case pid do
+      nil -> 0
```

Comment:
may be `-1` to show absence of device?

Reply:
Because MediaDb can be BackupDb or CargoDb. So my solution is check if there pids are existed and sum them up. So maybe 0 satisfy my thinking.

Reply:
I understand your idea an I will improve it